tests/integration/kubernetes/confidential_common.sh

```diff
@@ -106,4 +108,65 @@ function cleanup_loop_device(){
 	fi
 
 	sudo rm -f "$loop_file" >/dev/null 2>&1 || true
+}
+
+# This function creates pod yaml. Parameters
+# - $1: image reference
+# - $2: image policy file. If given, `enable_signature_verification` will be set to true
+# - $3: image registry auth.
+# - $4: guest components procs parameter
+# - $5: guest components rest api parameter
+# - $6: node
+function create_coco_pod_yaml() {
+    image=$1
+    image_policy=${2:-}
+	image_registry_auth=${3:-}
+	guest_components_procs=${4:-}
+	guest_components_rest_api=${5:-}
+	node=${6:-}
+
+	local CC_KBS_ADDR
+    export CC_KBS_ADDR=$(kbs_k8s_svc_http_addr)
+
+    kernel_params_annotation="io.katacontainers.config.hypervisor.kernel_params"
+	kernel_params_value=""
```

Comment:
minor comment: inconsistent whitespace indentation?

Reply:
Should be fixed now